client/src/components/common/ArtistHeaderSection.tsx

```diff
@@ -79,13 +78,11 @@ const ArtistActions = styled.div`
   }
 `;
 
-const ArtistHeaderSection: React.FC<{ artist: Artist; isManage?: boolean }> = ({
+const ArtistHeaderSection: React.FC<{ artist: Artist|undefined; isLoading: boolean; isManage: boolean }> = ({
```

Comment:
I did not! Prettier started reformatting everything with tabs so I turned it off :(

It could be that my installation is broken/outdated/idk - I'll try to get it working again now that I know you're also using it!

Reply:
Update: I've got it set up with `"useTabs": false, "tabWidth": 2, "trailingComma": "es5"` - which I think is closest to the rest of the codebase (although there are still a few formatting diffs in other files).

Would it be helpful if I PR this config into the repo so that it's enabled by default? (I've currently set it up locally with husky / `npx --no-install lint-staged` to run on commit, rather than using the vscode extension)

Reply:
Yeah I think a different PR for that would be helpful!